tests/serialize/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
> I would stay away from building tools if you can since the building is not very complex and maintaining the plugins and scripts for those is too much of a hassle imo

I agree.

> I would rather temporary place the scripts on the package.json.

If you can point me to some documentation that shows this multiple-package-dot-json-in-one-npm-package pattern as a supported feature of npm, I'd definitely consider it.

> I've rewritten a couple of the tests in mocha for now and made data.js into a class that return promises, they should all work async.

Nice! This is something I've really wanted to see happen (#151) but have not had the bandwidth for. I'd like to stay in sync with you on this, since the structure of the test suite will influence how we continue to add tests in the future.

> I'm writing all in typescript cause strong typing saves me some time, but I can just transpile them to es6 or directly in es5 if it's a problem.

I'm a fan of static typing too. That said we have no plans to introduce typescript into `rdflib.js`, so if you would like to get your test refactor merged it would have to be in es2015.

Also, `rdflib.js` unfortunately already contains checked-in source which was transpiled without edits from coffeescript and python (yes, python...). I wouldn't accept transpiled code unless it passed the turing test, if you see what I mean.

Thanks!